src/test/java/com/sugarcrm/candybean/runner/BlockListUnitTest.java

```diff
@@ -50,8 +50,9 @@ public void ignoreFail() throws Exception {
 	public void blockListFail() throws Exception {
 		fail();
 	}
-	
+
 	// To ensure VRunner @Test doesn't break
+	@Ignore
```

Comment:
@karanhuynh-sugarcrm, we're most likely ignoring this test because it was blocked in the previous tests in the same class, but do you know if we have opened a ticket to address this?